src/building/storage.c

```diff
@@ -15,12 +15,10 @@
 #define STORAGE_STATIC_BUFFER_SIZE 10
 #define STORAGE_CURRENT_BUFFER_SIZE (STORAGE_STATIC_BUFFER_SIZE + RESOURCE_MAX)
 
-typedef struct {
-    int id;
-    int in_use;
-    int building_id;
-    building_storage storage;
-} data_storage;
+#define FULL_STORAGE 32
+#define THREEQ_STORAGE 24
```

Comment:
Very small thing. This isn't your doing but it's time to fix.

```suggestion
#define THREE_QUARTERS_STORAGE 24
```

Reply:
Done.